pipelines/controller/src/main/java/com/google/fhir/analytics/ApiController.java

```diff
@@ -117,4 +126,38 @@ public ResponseEntity<InputStreamResource> download(@RequestParam(name = "path")
     headers.put("Content-type", Arrays.asList(MediaType.TEXT_PLAIN_VALUE));
     return new ResponseEntity<>(inputStreamResource, headers, HttpStatus.OK);
   }
+
+  @PostMapping("/testView")
```

Comment:
I see that using hypen(-) instead of camelCasing makes the URI more readable, check this https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

Reply:
Done.